src/PROPOSAL/PROPOSAL/particle/ParticleDef.h

```diff
@@ -103,6 +103,7 @@ enum class ParticleType : int {
     Monopole = 41,
     SMPPlus = 9500,
     SMPMinus = -9500,
+    Hadron = 81,
```

Comment:
__Codes 81–100 are reserved for generator-specific pseudoparticles and concepts.__
I like, that you keep the PDG code style 👍